core/server/api/authentication.js

```diff
@@ -10,7 +10,7 @@ const Promise = require('bluebird'),
     urlService = require('../services/url'),
     localUtils = require('./utils'),
     models = require('../models'),
-    spamPrevention = require('../web/middleware/api/spam-prevention'),
+    spamPrevention = require('../web/shared/middleware/api/spam-prevention'),
```

Comment:
Did you consider renaming this to `../shared/middlewares/...`? What are your thoughts?

Reply:
Yeah +1, I think should be `middlewares` as well, updating

Reply:
Cool :)